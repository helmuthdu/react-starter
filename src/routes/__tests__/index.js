import { shallow } from 'enzyme';
import React from 'react';
import { AppRouter } from '..';

describe('App component', () => {
  const history = {
    location: {
      path: '/'
    }
  };

  it('renders without crashing', () => {
    const wrapper = shallow(<AppRouter history={history} />);
    expect(wrapper).toMatchSnapshot();
  });
});
