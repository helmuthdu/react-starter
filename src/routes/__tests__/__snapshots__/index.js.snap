// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should match snapshot 1`] = `
<Connect(ConnectedRouterWithContext)
  history={
    Object {
      "location": Object {
        "path": "/",
      },
    }
  }
>
  <AppRoutes />
</Connect(ConnectedRouterWithContext)>
`;
